execute/plugin_test.go

```diff
@@ -416,8 +424,11 @@ func TestPlugin_ShouldAcceptAttestedReport_DoesNotDecode(t *testing.T) {
 		Return(cciptypes.ExecutePluginReport{}, fmt.Errorf("test error"))
 	p := &Plugin{
 		reportCodec: codec,
+		lggr:        logger.Test(t),
 	}
-	_, err := p.ShouldAcceptAttestedReport(context.Background(), 0, ocr3types.ReportWithInfo[[]byte]{})
+	_, err := p.ShouldAcceptAttestedReport(context.Background(), 0, ocr3types.ReportWithInfo[[]byte]{
+		Report: []byte("will not decode"), // faked out, see mock above
```

Comment:
Is this parameter needed or informational? I thought the mocked function would be enough.

Reply:
Its needed since I updated the `Reports` function to return if there is no report, since that will cause JSON decoding to error out.